packages/components/psammead-consent-banner/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.2.0 | [PR#1594](https://github.com/bbc/psammead/pull/1594) Update layout to flip correctly for right-to-left services |
```

Comment:
If we're adding `dir` as a required prop, this should be a breaking change, right?